src/Agent/Configuration/newrelic.config

```diff
@@ -21,13 +21,13 @@
 		stackTraceThreshold="500"
 		recordSql="obfuscated"
 		explainEnabled="false"
-		explainThreshold="500"/>
-	<crossApplicationTracer enabled="true"/>
+		explainThreshold="500" />
+	<distributedTracing enabled="true" />
```

Comment:
Is this required as it's the default now?

Reply:
This is how it's the default. :)